src/login/logind-action.c

```diff
@@ -346,6 +384,17 @@ int manager_handle_action(
                 return 1;
         }
 
+        if (handle == HANDLE_SECURE_ATTENTION_KEY) {
+                if (!is_edge)
+                        return 0;
+
+                if (!action_seat)
+                        return 0;
+
+                log_info("Requesting greeter on seat %s...", action_seat);
```

Comment:
the function already writes a structured log message, at LOG_INFO saying pretty much the same thing. Hence, please drop this line